recipes/libpcap/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import AutoToolsBuildEnvironment, tools, ConanFile
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibPcapConan(ConanFile):
+    name = "libpcap"
+    version = "1.9.1"
```

Comment:
If I do delete this I can't run: conan create all libpcap/1.9.1
But for sure it shouldn't be present here.

Reply:
`conan create all libpcap/1.9.1@` should work

Reply:
That wasn't obvious at all, thanks!